src/ZendeskApi.Client/Resources/HttpRequestExtensions.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
```

Comment:
The HttpChannel class I felt was a bit of a mess. now if you want to customise a HttpClient, you can just implement IZendekApiClient and return a new HttpClient. I dont see why someone would want to customise that. Is there a usecase you can think of or have come across?